src/Analyzer.Utilities.UnitTests/JsonLineNumberResolverTests.cs

```diff
@@ -342,5 +342,193 @@ public void Constructor_NullTemplateContext_ThrowsException()
         {
             new JsonLineNumberResolver(null);
         }
+
+        [TestMethod]
+        [ExpectedException(typeof(Exception))]
+        public void ResolveLineNumber_NullRootTemplateContext_ThrowsException()
```

Comment:
Update this test to assign a valid template to `OriginalTemplate`, and explicitly assign null to the parent context.  As this test is currently written, I don't think it actually tests what you intended, because it will have the same behavior as test "ResolveLineNumber_OriginalTemplateIsNull_ThrowsException" above.

Reply:
(i.e. the exception thrown will be because OriginalTemplate is null, not because the parent context is null.)